databuilder/extractor/neo4j_search_data_extractor.py

```diff
@@ -44,8 +46,11 @@ def init(self, conf):
         self.conf = conf
 
         # extract cypher query from conf, if specified, else use default query
-        self.cypher_query = conf.get_string(Neo4jSearchDataExtractor.CYPHER_QUERY_CONFIG_KEY,
-                                            Neo4jSearchDataExtractor.DEFAULT_NEO4J_CYPHER_QUERY)
+        if Neo4jSearchDataExtractor.CYPHER_QUERY_CONFIG_KEY in conf:
+            self.cypher_query = conf.get_string(Neo4jSearchDataExtractor.CYPHER_QUERY_CONFIG_KEY)
```

Comment:
do you have an example when user provides the cypher query himself?

Reply:
I think custom query can start with default query which is defined above.